python/test_infra/ttir_builder.py

```diff
@@ -267,24 +267,26 @@ def eltwise_proxy(
             len(stack) > 0
         ), "Top of callstack to builder funcs must be outside this file"
 
+        id = self.get_next_global_id()
+
+        # FIXME: this should be a `Location.file`, but for some reason it causes
+        # strange decomposition inheritance behaviour that breaks using this as
+        # a key into the golden map
+        loc = Location.name(
+            stack[0].filename + ":" + str(stack[0].lineno) + ":id(" + str(id) + ")"
+        )
```

Comment:
@ctodTT Can you elaborate more abt this? There might be a way to fix it if it's based on the way things are being parsed.